src/components/Header/Header.tsx

```diff
@@ -0,0 +1,12 @@
+import { useState } from "react"
+import { fetchHeader } from "../../utils/strapiAPIs"
+
+export default function Header() {
+  const [title, setTitle] = useState<string>(fetchHeader())
```

Comment:
No need to use state here, since you're not manipulating the `title` variable, just grab the title from the hook.

`const title = fetchHeader();`